polkadot/node/core/pvf/common/src/pvf.rs

```diff
@@ -115,7 +115,7 @@ impl fmt::Debug for PvfPrepData {
 	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
 		write!(
 			f,
-			"Pvf {{ code, code_hash: {:?}, executor_params: {:?}, prep_timeout: {:?} }}",
+			"Pvf {{ code_hash: {:?}, executor_params: {:?}, prep_timeout: {:?} }}",
```

Comment:
nit: Hmm, maybe this, to make it more obvious that a field is omitted:

```suggestion
			"Pvf {{ code: [...], code_hash: {:?}, executor_params: {:?}, prep_timeout: {:?} }}",
```